tests/Feature/Proxy/ShopProxyTest.php

```diff
@@ -64,6 +64,10 @@ public function testAccessors()
         /** @var AddressInterface $returnAddress */
         $returnAddress = $addressBuilder->getMock();
         $this->assertEquals($returnAddress, $this->shopProxy->setReturnAddress($returnAddress)->getReturnAddress());
+
+        /** @var AddressInterface $senderAddress */
+        $senderAddress = $this->getMockBuilder(AddressInterface::class)->getMock();
```

Comment:
Can't you just use the $addressBuilder, instead of $this->getMockBuilder(AddressInterface::class)?

For consistency purposes. :yum:

Reply:
Oh, yeah, I must've copied it from somewhere else.